src/frequenz/sdk/actor/power_distributing/_component_status/_ev_charger_status_tracker.py

```diff
@@ -0,0 +1,204 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Background service that tracks the status of an EV charger."""
+
+
+import asyncio
+import logging
+from datetime import datetime, timedelta, timezone
+
+from frequenz.channels import Receiver, Sender, select, selected_from
+from frequenz.channels.timer import SkipMissedAndDrift, Timer
+from frequenz.client.microgrid import (
+    EVChargerCableState,
+    EVChargerComponentState,
+    EVChargerData,
+)
+from typing_extensions import override
+
+from frequenz.sdk.microgrid import connection_manager
+
+from ..._background_service import BackgroundService
+from ._blocking_status import BlockingStatus
+from ._component_status import (
+    ComponentStatus,
+    ComponentStatusEnum,
+    ComponentStatusTracker,
+    SetPowerResult,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+class EVChargerStatusTracker(ComponentStatusTracker, BackgroundService):
+    """Status tracker for EV chargers.
+
+    It reports an EV charger as `WORKING` when an EV is connected to it,
+    and power can be allocated to it, and `NOT_WORKING` otherwise.
+
+    If it receives a power assignment failure from the PowerDistributor,
+    when the component is expected to be `WORKING`, it is marked as
+    `UNCERTAIN` for a specific interval, before being marked `WORKING`
+    again.
+    """
+
+    @override
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        component_id: int,
+        max_data_age: timedelta,
+        max_blocking_duration: timedelta,
+        status_sender: Sender[ComponentStatus],
+        set_power_result_receiver: Receiver[SetPowerResult],
+    ) -> None:
+        """Create an `EVChargerStatusTracker` instance.
```

Comment:
```suggestion
        """Initialize this instance.
```
Super minor and just mentioning it because you used "Initialize" in other `__init__()`s.